esphomeyaml/cookbook/h801.rst

```diff
@@ -0,0 +1,96 @@
+H801 RGBW LED controller
+========================
+
+The H801 is pretty afordable and easy to hack and adapt to your needs. It can be found on `Aliexpress <https://s.click.aliexpress.com/e/bbnUDBZW>`__
+`Here <http://tinkerman.cat/closer-look-h801-led-wifi-controller/>`__ is an article about the hardware for those who have an interest for more details about the board.
+It can be used as a RGB, RGBW, RGB dual white or even control 5 individual monochromatic strips if you want or combinations of these.
+
+.. figure:: images/h801.jpg
+ :align: center
+ :width: 80.0%
+
+For esphomelib, you can then use the :doc:`RGBW </esphomeyaml/components/light/rgbw>` 
+and the :doc:`ESP8266 Software PWM output </esphomeyaml/components/output/esp8266_pwm>` components using below configuration:
+
+
+.. code-block:: yaml
+    esphomeyaml:
+      name: h801test
+      platform: ESP8266
+      board: d1_mini

```

Comment:
There's not a d1_mini inside of this device. I think it is a `esp01_1m`. That would also explain why the OTA upload is not working (if you set the incorrect model OTA thinks the binary is too big and aborts)